features/checkout/addressing_order/addressing_order.feature

```diff
@@ -0,0 +1,38 @@
+@checkout_addressing
```

Comment:
`features/checkout/addressing_order/addressing_order.feature` - repeated `addressing_order` looks weird. Maybe we should name the feature `filing_shipping_and_billing_address_details`?